### PR TITLE
WIP: Add Django application and OpenShift skeleton

### PR DESCRIPTION
**Pull request by shaded-enmity**

### CODE REVIEW

**Comment by shaded-enmity** [comment]
jenkins: ok to test

File: ansible_roles/mezzanine/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+- name: Make sure manage.py is executable
+  file:
+    path: /opt/blog/manage.py
+    mode: 'ug+x'
+
+- name: Initialize the database
+  django_manage:
+    command: createdb --noinput
+    app_path: /opt/blog
+  #register: result
```
Comment (vmindru) [inline ansible_roles/mezzanine/tasks/main.yml]: do we really need this comments here?

File: demo/vmdefs/centos7-mezzanine/Vagrantfile [hunk 2]
```diff
@@ -0,0 +1,29 @@
+VAGRANTFILE_API_VERSION = "2"
```
Comment (vmindru) [inline demo/vmdefs/centos7-mezzanine/Vagrantfile]: let's create a similar concept like for http vagrant-boxes-enabled and vagrant-boxes-disabled, or some specific way where we can easily specify what boxes we want to start. Right now we would be starting 4 vagrant boxes and we would need cca 16GB RAM per instance.

Comment (shaded-enmity) [inline demo/vmdefs/centos7-mezzanine/Vagrantfile]: Good point, I've added a `enabled` directory and symlinked the phpMyAdmin, Drools and Target VMs there (+ update to `{start,stop,destroy}_vms.sh`).

**Comment by shaded-enmity** [comment]
Example output of metadata running against a Django app:

```
[
    {
        "path": "/opt/blog",
        "data": {
            "cache": {
                "default": {
                    "LOCATION": "127.0.0.1:11211",
                    "BACKEND": "django.core.cache.backends.memcached.MemcachedCache"
                }
            },
            "db": {
                "default": {
                    "ENGINE": "django.db.backends.postgresql_psycopg2",
                    "AUTOCOMMIT": true,
                    "ATOMIC_REQUESTS": false,
                    "NAME": "db",
                    "CONN_MAX_AGE": 0,
                    "TIME_ZONE": null,
                    "PORT": "",
                    "HOST": "127.0.0.1",
                    "USER": "meza",
                    "TEST": {
                        "COLLATION": null,
                        "CHARSET": null,
                        "NAME": null,
                        "MIRROR": null
                    },
                    "PASSWORD": "meza007",
                    "OPTIONS": {}
                }
            }
        },
        "unit": {
            "unit_name": "mezzanine-blog.service",
            "description": "Mezzanine Blog",
            "active_state": "active",
            "object_path": "/org/freedesktop/systemd1/unit/mezzanine_2dblog_2eservice",
            "sub_state": "running",
            "exec_start": [
                "/bin/python",
                "/opt/blog/manage.py",
                "runserver",
                "--insecure",
                "0.0.0.0:80"
            ],
            "followers": "",
            "load_state": "loaded"
        },
        "settings": [
            "/opt/blog/blog/settings.py",
            "/opt/blog/blog/local_settings.py"
        ]
    }
]
```

**Comment by vinzenz** [comment]
Closing this PR - As this is no longer applicable to this repository

**Pull request closed without merging**